installer/roles/local_docker/templates/docker-compose.yml.j2

```diff
@@ -7,7 +7,7 @@ services:
     container_name: awx_web
     depends_on:
       - redis
-      - memcached
+      - django-redis
```

Comment:
```suggestion
```

I think we can just delete this line. This isnt a python dependency, but rather a docker-compose dependency. The `redis` above should be all we need.